doctors_surgery.html

```diff
@@ -45,13 +45,13 @@ <h3><strong><em>COVID RESTRICTION UPDATE</em></strong></h3>
     </section>
 
     <ul>
-        <li><a href="#">GET HELP NOW</a></li>
+        <li><a href="#">Get help now</a></li>
```

Comment:
With these links, I would probably have them rendered as uppercase by applying styling. If the HTML contains all uppercase letters then I think a screen reader will attempt to turn them in to an acronym, rather than viewing them as words.

Reply:
Wow I never knew that! I've often wondered what makes people choose to upper case things in the CSS!